test/m4/appframework/appframework_test.go

```diff
@@ -258,7 +258,7 @@ var _ = Describe("m4appfw test", func() {
 			uploadedApps = append(uploadedApps, uploadedFiles...)
 
 			// Wait for the poll period for the apps to be downloaded
-			time.Sleep(2 * time.Minute)
+			time.Sleep(4 * time.Minute)
```

Comment:
this is a timer between app polling. Not related to the timer for phase. Those timer values are  here (https://github.com/splunk/splunk-operator/blob/master/test/testenv/testenv.go#L64)

Reply:
I have updated the information , this is temporary fix, just to see if this fixes the issue,